src/styles/components/statusbar.less

```diff
@@ -0,0 +1,59 @@
+.status-bar {
+  display: inline-flex;
+  background-color: @gray;
+  margin-right: 0;
+  height: @base-spacing-unit * 1/3;
+  width: 114px;
+  max-width: 100%;
+
+  .bar {
+    background-color: currentColor;
+    border-left: 1px solid @black;
+
+    &:first-child {
+      border-left: none;
+    }
+  }
+}
+
+.status-bar-column {
+  width: 138px;
+}
+
+.status-bar-wrapper{
+  display: inline-block;
+  .running-state {
+    color: @white;
+  }
+  .deploying-state {
+    color: @yellow;
+  }
+}
+
+.dot {
```

Comment:
If these are `dot`-styles, then let's move them to `badges.less` or even better it's own `dot.less` styles